app-misc/deark/deark-1.6.5.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Utility to extract data from various file formats"
+DESCRIPTION="Deark is a command-line utility that can decode certain types of files, and either: \
+\
+1. convert them to a more-modern or more-readable format; or \
+2. extract embedded files from them"
+HOMEPAGE="https://entropymine.com/deark/ https://github.com/jsummers/deark"
+COPYRIGHT="2023 Jason Summers"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/jsummers/deark/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="2aa29675c0b326c5242da2c5e4b1d7746f2aad7bcc9bfefa73448e580e9fae2c"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	deark$secondaryArchSuffix = $portVersion
+	cmd:deark = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	help2man$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:help2man
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	make
+	make man
+}
+
+INSTALL()
+{
+	mkdir -p $binDir
```

Comment:
Better install in $prefix/bin (otherwise you need `cmd:deark$secondaryArchSuffix` in PROVIDES).

Reply:
Looking at the Makefile, there is an option to set this?:
```
	make DEARK_INSTALLDIR=$prefix/bin
```
https://github.com/jsummers/deark/blob/master/Makefile#L143

Reply:
Is it good practice to use `$secondaryArchSuffix`? Or does it not matter, and INSTALL just needs to be changed to suit?

Reply:
For packages `not building with gcc2` we prefer to use $prefix/bin (you can find examples when looking for `commandBinDir` here) for the binaries on 32bit, this resolves to have to use $secondaryArchSuffix for binaries, libraries should have it `though` as that path is important for them.